aim/config.py

```diff
@@ -32,18 +32,6 @@
 
 LOG = logging.getLogger(__name__)
 
-global_opts = [
```

Comment:
Heads up: removing these options will break the AIM ML2 driver (driver initialize and UTs). Please be ready with follow-on patches to the ML2 driver.